ansible/influxdb-meta/Vagrantfile

```diff
@@ -2,41 +2,49 @@
 # vi: set ft=ruby :
 
 Vagrant.configure(2) do |config|
-  config.vm.box = "ubuntu/trusty64"
-  # config.vm.box = "relativkreativ/centos-7-minimal"
-  # config.vm.box = "puppetlabs/centos-6.6-64-nocm"
-  # config.vm.box = "debian/jessie64"
+  config.vm.box = ENV["VAGRANT_VM_BOX"] ||  "ubuntu/trusty64"
+
+  BOX_COUNT      = ENV["VAGRANT_BOX_COUNT"] ? ENV["VAGRANT_BOX_COUNT"].to_i : 3
+  MEMORY         = ENV["VAGRANT_MEMORY"] || 256
+  CPUS           = ENV["VAGRANT_CPUS"] || 1
+  HTTP_BIND_PORT = 8091
+
+  leader = ""
+  members = Array.new
 
-  BOX_COUNT = 1
   (1..BOX_COUNT).each do |machine_id|
-    config.vm.define "influx#{machine_id}" do |machine|
-      machine.vm.hostname = "influx#{machine_id}"
-      machine.vm.network "private_network", type: "dhcp"
-      # machine.vm.network "public_network"
-      # machine.vm.network "public_network", :bridge => 'en0: Wi-Fi (AirPort)'
+    primary      = machine_id == 1 ? true : false
+    host_postfix = "%03d" % machine_id
+    hostname     = "metanode-#{host_postfix}"
+    leader       = hostname if machine_id == 1
```

Comment:
I like the idea of being able to set a leader, but this doesn't actually work in practice. With raft, the leader will vary as time goes on and isn't a static variable. It will usually end up working in the Vagrant case since you always add the `1` node first, however the longer the cluster is up the higher the chance that the leader will change.